tests/test_bug_features.py

```diff
@@ -266,3 +268,16 @@ def test_is_same_target_milestone(test_data, expected):
 @pytest.mark.parametrize("test_data, expected", FIRST_AFFECTED_PARAMS)
 def test_is_first_affected_same(test_data, expected):
     assert is_first_affected_same()(test_data) == expected
+
+
+BUG_EXTRACTOR_PARAMS = [
+    ([has_str(), has_str()], [fileref(), url()]),
+    ([has_str(), has_url()], [fileref(), fileref()]),
+]
+
+
+@pytest.mark.parametrize("feature_extractors,cleanup_functions", BUG_EXTRACTOR_PARAMS)
+def test_BugExtractor(feature_extractors, cleanup_functions):
+    BugExtractor([has_str(), has_url()], [fileref(), url()])
```

Comment:
You are running this twice because of the parametrization. Just remove the parametrization and run the checks directly in the test